openlibrary/plugins/openlibrary/js/goodreads_import.js

```diff
@@ -2,41 +2,37 @@ import Promise from 'promise-polyfill';
 
 export function initGoodreadsImport() {
 
-    var l, elem, count, prevPromise;
+    var count, prevPromise;
 
     $(document).on('click', 'th.toggle-all input', function () {
         var checked = $(this).prop('checked');
-        if (!checked) {
-            $(this).prop('checked', false);
-            $('input.add-book').each(function () {
-                $(this).prop('checked', false);
-            });
-        }
-        else {
-            $(this).prop('checked', true)
-            $('input.add-book').each(function () {
-                $(this).prop('checked', true);
-            });
-        }
-        l = $('.add-book[checked*="checked"]').length;
+        $('input.add-book').each(function () {
+            $(this).prop('checked', checked);
+            if (checked) {
+                $(this).attr('checked', 'checked');
+            }
+            else {
+                $(this).removeAttr('checked');
+            }
+        });
+        const l = $('.add-book[checked*="checked"]').length;
         $('.import-submit').attr('value', `Import ${l} Books`);
     });
 
     $(document).on('click', 'input.add-book', function () {
-        var checked = $(this).prop('checked');
-        if (!checked) {
-            $(this).prop('checked', false);
+        if ($(this).prop('checked')) {
+            $(this).attr('checked', 'checked');
         }
         else {
-            $(this).prop('checked', true);
+            $(this).removeAttr('checked');
         }
-        l = $('.add-book[checked*="checked"]').length;
+        const l = $('.add-book[checked*="checked"]').length;
         $('.import-submit').attr('value', `Import ${l} Books`);
     });
 
     function func1(value) {
-        l = $('.add-book[checked*="checked"]').length;
-        elem = document.getElementById('myBar');
+        var l = $('.add-book[checked*="checked"]').length;
+        var elem = document.getElementById('myBar');
```

Comment:
```suggestion
        const l = $('.add-book[checked*="checked"]').length;
        const elem = document.getElementById('myBar');
```
These are not reassigned, so we can declare as a constant.